src/client/components/EducationHistoryTile.js

```diff
@@ -23,10 +23,10 @@ const EducationHistoryTile = ({
       <Dates>{displayStartEndDates(educationHistoryData)}</Dates>
     </div>
     <Icons>
-      <Button onClick={() => openModal()}>
+      <Button onClick={openModal}>
         <IconPencil width="16px" height="16px" color={theme.accent} />
       </Button>
-      <Button onClick={() => removeItem()}>
+      <Button onClick={removeItem}>
```

Comment:
I love cleanup like this. Thanks!

With big PRs like this one, though, any amount of noise makes it harder to review. I'd love to see stuff like this and the linting cleanup in separate, focused PRs. 

Deciding which changes get grouped into which PR is an art form, and we don't have to be super pedantic about it. But it really does help the review process, to think about whether a given change fits the theme of the rest of the PR.

Reply:
Point seriously taken!